lib/notifications/display.dart

```diff
@@ -169,8 +165,6 @@ class NotificationDisplayManager {
         iconBitmap: await _fetchBitmap(data.senderAvatarUrl))));
 
     await _androidHost.notify(
-      // TODO the notification ID can be constant, instead of matching requestCode
-      //   (This is a legacy of `flutter_local_notifications`.)
       id: notificationIdAsHashOf(conversationKey),
```

Comment:
This TODO suggests we can make the notification ID a constant, instead of this hash. Can we do that now, or is there still an obstacle to it?